walk.go

```diff
@@ -393,6 +393,9 @@ func (s *PromQLSmith) walkVariadicFunctions(expr *parser.Call) {
 	case "round":
 		expr.Args[0] = s.Walk(expr.Func.ArgTypes[0])
 		expr.Args[1] = &parser.NumberLiteral{Val: float64(s.rnd.Intn(10))}
+	case "info":
+		expr.Args[0] = s.Walk(expr.Func.ArgTypes[0])
+		expr.Args[1] = s.Walk(expr.Func.ArgTypes[1])
```

Comment:
The second argument has to be a vector selector.
If you do `s.Walk(<vector-type>)` it will generate an expression which has a result value type to be vector but it can be a different expression like function.

Reply:
I think we can have a dedicated `walkInfo` function. The second parameter can be either skipped or kept based on rand.